pkg/pipeline/transform/transform_network.go

```diff
@@ -141,6 +142,13 @@ func (n *Network) Transform(inputEntry config.GenericMap) (config.GenericMap, bo
 					}
 				}
 			}
+		case api.NetworkDecodeTCPFlags:
+			if anyFlags, ok := outputEntry[rule.DecodeTCPFlags.Input]; ok && anyFlags != nil {
+				if flags, ok := anyFlags.(uint16); ok {
+					flags := util.DecodeTCPFlags(flags)
+					outputEntry[rule.DecodeTCPFlags.Output] = strings.Join(flags, ",")
```

Comment:
Why not using a string array here ?

Reply:
done https://github.com/netobserv/flowlogs-pipeline/pull/747/commits/974b3f54d5505fa13c91323816ffadafc53c86aa